server/src/templates/agents.html

```diff
@@ -17,14 +17,18 @@ <h1 class="p-heading--3">
     <button type="reset" class="p-search-box__reset"><i class="p-icon--close"></i></button>
     <button type="submit" class="p-search-box__button"><i class="p-icon--search"></i></button>
 </form>
-<table aria-label="Agents table" class="p-table--mobile-card">
+<table aria-label="Agents table" class="p-table--mobile-card" id="agentsTable">
     <thead>
         <tr>
-            <th aria-sort="none" class="has-overflow" style="width: 40pt"></th>
-            <!-- Added an empty header for the warning icon column -->
-            <th aria-sort="none">Name</th>
-            <th aria-sort="none">State</th>
-            <th aria-sort="none">Updated At</th>
+            <th aria-sort="none" class="has-overflow" style="width: 80pt" use-outcome-sort id="agentsOutcomeHeader">
```

Comment:
Hey, great work, I just have one comment in regards to the icons. You should add the `p-table__cell--icon-placeholder` on the `th` and `td` elements that have icons; this will make the icon display outside the column

See https://vanillaframework.io/docs/base/tables#icons

```suggestion
            <th aria-sort="none" class="p-table__cell--icon-placeholder has-overflow" style="width: 80pt" use-outcome-sort id="agentsOutcomeHeader">
```

Reply:
The `td` element seems to be at line 38. I'm not sure how the icon placeholder behavior and the span and tooltip you have will interact with each other.

https://github.com/canonical/testflinger/blob/f8f32c38772361ad4c84328191120c1e3edfacd7/server/src/templates/agents.html#L38

Reply:
I would also suggest you remove the `style="width: 80pt`, as it is generally bad practice to hard-code these styles into elements, and a hard-coded width can lead to some "fun" results...

![image](https://github.com/user-attachments/assets/c55afc74-d526-4485-bce7-1cef9cd1caaf)

Reply:
Decided not to add the icons on the outside, as they are actually more informational/related to the column data, so they are fine to keep within the column.